package.json

```diff
@@ -8,12 +8,11 @@
     "docker:start": "docker-compose up -d",
     "docker:stop": "docker-compose down",
     "pretest": "npm run lint",
-    "test": "npm run docker:start && sleep 10 && nyc ava test/unit test/integration; npm run docker:stop",
-    "test:ci": "nyc ava test/unit test/integration",
+    "test": "npm run docker:start && sleep 10 && . ./init-environment.sh && nyc ava test/unit test/integration; npm run docker:stop",
```

Comment:
This `. <file>` syntax is equivalent to `source <file>` in bash. In dash, it is not. `source` doesn't exist in dash. `. <file>` is more portable.